addons/escoria-core/game/core-scripts/esc/commands/print.gd

```diff
@@ -23,5 +23,6 @@ func configure() -> ESCCommandArgumentDescriptor:
 
 # Run the command
 func run(command_params: Array) -> int:
-	print(command_params[0])
+	# Replace the names of any globals in "{ }" with their value
+	print(escoria.logger._replace_globals(command_params[0]))
```

Comment:
If you're going to use `_replace_globals`, then it'll either need to be renamed to follow a public method convention, or you'll have to make a public method that defers to `_replace_globals` in the same class.